src/navbar/navbar-items.styles.tsx

```diff
@@ -39,9 +46,8 @@ export const MobileWrapper = styled.ul`
 // =============================================================================
 export const LinkItem = styled.li`
     position: relative;
-    display: flex;
+    display: contents;
```

Comment:
can this be reverted to `flex`? would prefer to avoid `contents` as it makes it harder to reason about the css + has poor accessibility support in older browsers. can try moving LinkIndicator as a sibling of MenuItemContainer instead to get the absolute positioning